### ISSUE TITLE
has_specified_input axioms & analyte/measurand roles

### ISSUE THREAD

**Issue by johnwjudkins** [obi-ontology/obi#1642]
The assay template analyte and measurand columns construct axioms thusly:
`C (has_specified_input some %) and (realizes some ('analyte role' and ('inheres in' some %)))`
I remember hearing a suggestion that we drop the `has_specified_input`axiom, because the analyte/measurand is not necessarily present in the evaluant. I didn't see a tracker item addressing this.

**Comment by johnwjudkins** [comment]
2023-02-06 Will remove mentioned axiom (as discussed for #1615).

**Comment by jamesaoverton** [comment]
I was on that call and participated in the discussion, but this was not my understanding of the outcome. I thought you were going to remodel your organism detection terms, not all the assays in the assay template. PR #1656 is changing thousands of axioms.

**Comment by johnwjudkins** [comment]
@jamesaoverton I thought there was agreement, but maybe I should have been more clear that I intended to change the axiom-building column headers in assays.tsv. I see your point about the number of changes. It's possible _some_ assay terms _should_ keep the has_specified_input axiom for their analyte/measurand.

Here's what I propose to do for #1656:

1. revert the change in the commit I linked in this tracker item (09fd88e), which would restore the has_specified_input axiom to the analyte and measurand column headers of assays.tsv.
2. remake assays.owl
3. comment on the PR to notify Hector of the change, since he is assigned to review the PR.

Is that OK?
